ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo/Byron.hs

```diff
@@ -98,11 +98,16 @@ checkDelegationSubject vk = (== vk) . Cardano.Delegation.delegateVK
 -- | See chapter 4.1 of
 --   https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/byronChainSpec/latest/download-by-type/doc-pdf/blockchain-spec
 mainnetPBftSignatureThreshold :: PBftSignatureThreshold
-mainnetPBftSignatureThreshold = PBftSignatureThreshold 7
+mainnetPBftSignatureThreshold = PBftSignatureThreshold 0.22
```

Comment:
:relieved:

Reply:
I didn't expect that people do review commit-by-commit -- and that was an early typo : -)